src/main/java/Transaction.java

```diff
@@ -0,0 +1,71 @@
+import java.math.BigDecimal;
+import java.sql.ResultSet;
+import java.time.LocalDate;
+
+class Transaction {
+
+    enum Option {
```

Comment:
jak chcemy to jako enuma, to na pewno nie w klasie Transaction. Powinniśmy zrobić osobną klasę na to.